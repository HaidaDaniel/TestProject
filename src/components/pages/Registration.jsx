/** @format */
import Header from '../Header'
import { Container, Row, Col } from 'react-bootstrap'
import RegistrationBody from '../RegistrationBody'

function Registration() {
  return (
    <Container>
      <Row>
        <Col>
          <Header></Header>
        </Col>
      </Row>
      <Row>
        <Col>
          <RegistrationBody></RegistrationBody>
        </Col>
      </Row>
    </Container>
  )
}
export default Registration

Registration.propTypes = {}
