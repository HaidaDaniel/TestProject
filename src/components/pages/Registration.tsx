/** @format */
import { Container, Row, Col } from 'react-bootstrap'
import { RegistrationBody, Header } from '../'

const Registration = () => (
  <Container>
    <Row>
      <Col>
        <Header />
      </Col>
    </Row>
    <Row>
      <Col>
        <RegistrationBody />
      </Col>
    </Row>
  </Container>
)

export default Registration
