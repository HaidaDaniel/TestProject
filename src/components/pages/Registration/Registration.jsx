/** @format */

import RegistrationBody from '../../RegistrationBody/RegistrationBody'

function Registration() {
  return <RegistrationBody></RegistrationBody>
}
export default Registration
