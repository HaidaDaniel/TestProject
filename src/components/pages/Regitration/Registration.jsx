/** @format */

import Header from '../../Header/Header'
import { Container } from 'react-bootstrap'

function Regisrtation() {
  return <></>
}
export default Regisrtation
