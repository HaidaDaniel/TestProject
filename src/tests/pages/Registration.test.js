import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

<<<<<<<< HEAD:src/tests/components/pages/Registration.test.js
import Registration from '../../../components/pages/Registration';

import wrapComponentInAllProviders from '../../helpers/wrapComponentInAllProviders';


describe('Registration Component', () => {
    it('should render registration page', () => {
========
import wrapComponentInAllProviders from '../../tests/helpers/wrapComponentInAllProviders';
import Registration from '../../pages/Registration';

describe('Registration Component', () => {
    it('should renders registration page', () => {
>>>>>>>> origin/newTest:src/tests/pages/Registration.test.js
        render(
            wrapComponentInAllProviders(
                <Registration />));

        expect(document.body).toMatchSnapshot();

        const headerElement = screen.getByText('Shop');
        expect(headerElement).toBeInTheDocument();

        const registrationBodyElement = screen.getByText('Registration Form');
        expect(registrationBodyElement).toBeInTheDocument();

    });
});
