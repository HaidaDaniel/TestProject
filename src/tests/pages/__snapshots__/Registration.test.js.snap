// Jest Snapshot v1, https://goo.gl/fbAQLP

<<<<<<<< HEAD:src/tests/__snapshots__/components/pages/Registration.snap.js
exports[`Registration Component should render registration page 1`] = `
========
exports[`Registration Component should renders registration page 1`] = `
>>>>>>>> origin/newTest:src/tests/pages/__snapshots__/Registration.test.js.snap
<body>
  <div>
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col"
        >
          <nav
            class="sc-jrkPcm hELBzT navbar navbar-expand-lg navbar-light"
          >
            <a
              class="sc-kDnyCx ctmCGs mx-4"
              href="/"
            >
              Shop
            </a>
            <button
              aria-controls="basic-navbar-nav"
              aria-label="Toggle navigation"
              class="navbar-toggler collapsed"
              type="button"
            >
              <span
                class="navbar-toggler-icon"
              />
            </button>
            <div
              class="sc-ipMvLY dWioSl justify-content-end mx-2 navbar-collapse collapse"
              id="basic-navbar-nav"
            >
              <div
                class="ml-auto mx-2 navbar-nav"
              >
                <a
                  class="nav-link"
                  href="/registration"
                >
                  Registration
                </a>
                <a
                  class="nav-link"
                  href="/login"
                >
                  Login
                </a>
              </div>
            </div>
          </nav>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col"
        >
          <div
            class="row"
          >
            <div
              class="sc-grXZZQ kHhxCO"
            >
              <div
                class="sc-dkjKgF bsklqQ col-md-6 col-12"
              >
                <h3
                  class="sc-hLclGa EkSqt"
                >
                  Registration Form
                </h3>
                <form
                  action="#"
                >
                  <div
                    class="sc-eDnVMP cCBekH"
                  >
                    <label
                      class="sc-jSwlEQ gMDmUn form-label"
                    >
                      Email address
                    </label>
                    <input
                      class="sc-gKHVLF kxfaKY form-control"
                      name="email"
                      placeholder="Email address"
                      type="email"
                      value=""
                    />
                  </div>
                  <div
                    class="sc-eDnVMP cCBekH"
                  >
                    <label
                      class="sc-jSwlEQ gMDmUn form-label"
                    >
                      First name
                    </label>
                    <input
                      class="sc-gKHVLF kxfaKY form-control"
                      name="firstName"
                      placeholder="First name"
                      value=""
                    />
                  </div>
                  <div
                    class="sc-eDnVMP cCBekH"
                  >
                    <label
                      class="sc-jSwlEQ gMDmUn form-label"
                    >
                      Last name
                    </label>
                    <input
                      class="sc-gKHVLF kxfaKY form-control"
                      name="lastName"
                      placeholder="Last name"
                      value=""
                    />
                  </div>
                  <div
                    class="sc-eDnVMP cCBekH"
                  >
                    <label
                      class="sc-jSwlEQ gMDmUn form-label"
                    >
                      Password
                    </label>
                    <input
                      class="sc-gKHVLF kxfaKY form-control"
                      name="password"
                      placeholder="Password"
                      type="password"
                      value=""
                    />
                  </div>
                  <div
                    class="sc-eDnVMP cCBekH"
                  >
                    <label
                      class="sc-jSwlEQ gMDmUn form-label"
                    >
                      Confirm Password:
                    </label>
                    <input
                      class="sc-gKHVLF kxfaKY form-control"
                      name="confirmPassword"
                      placeholder="Confirm Password"
                      type="password"
                      value=""
                    />
                  </div>
                  <button
                    class="btn btn-primary"
                    type="submit"
                  >
                    Registration
                  </button>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
